src/korean.js

```diff
@@ -21,13 +21,21 @@ class Korean {
     if (word[wordLength - 2] === '하') {
         conjugate = word.slice(0, wordLength - 2);
         return conjugate.concat('해');
-    } else {
+    } else if (word[wordLength -2] === '르') {
```

Comment:
I just wanted to clarify the grammar rule a bit more.  This rule's condition says the word should have "르" AND the vowel before that should be either ㅗ or ㅜ.

I understand this might be difficult to understand due to the Korean language. If so, please leave it as is and I'll fix it in another MR.